src/config.js

```diff
@@ -204,8 +204,16 @@ global.config = {
   mineral: {
     enabled: true,
     storage: 100000,
-    minAmount: 5000,
-    minAmountForMarket: 100000,
+    minAmount: 5000
+  },
+
+  market: {
+    minAmount: 100000,
+    energyCreditEquivalent: 1,
+    trySellOrders: true,
+    sellOrderMaxAmount: 100,
+    sellOrderReserve: 2000,
+    sellOrderPriceMult: 5
```

Comment:
I guess `Mult` means `Multiplicator`, mind writing the full word, doesn't cost anything and easier to grasp the meaning